std-bits/base/src/main/java/org/enso/base/CurrentEnsoProject.java

```diff
@@ -1,19 +1,41 @@
 package org.enso.base;
 
+import org.enso.base.polyglot.EnsoMeta;
+import org.graalvm.polyglot.Value;
+
 /** A Java interface to the `Enso_Project` type. */
 public final class CurrentEnsoProject {
   private final String name;
   private final String namespace;
```

Comment:
Use a record class?

Reply:
`record` implies public constructor, but I wanted to keep the constructor private, so the only way to get an instance of this class is through `CurrentEnsoProject.get()`.